queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * from animals WHERE name LIKE '%mon';
+SELECT * from animals WHERE TO_CHAR(date_of_birth, 'yyyy') BETWEEN '2016' and '2019';
+SELECT name from animals where neutered = false and escape_attempts > 3;
+SELECT date_of_birth from animals where name IN ('Agumon', 'Pikachu'); 
+SELECT name, escape_attempts from animals where weight_kg > 10.5;
+SELECT name from animals where neutered = true;
+SELECT * from animals WHERE name LIKE '%Gabumon';
```

Comment:
- [ ] According to the requirements, we need to find all animals not named Gabumon. So I suggest using `not LIKE` or `!=`  or  `<>`💯

![image](https://user-images.githubusercontent.com/45231276/196313518-1e454256-4d99-4c12-9184-5e03791dea86.png)